app/controllers/posts_controller.rb

```diff
@@ -1,6 +1,6 @@
 class PostsController < ApplicationController
   def index
-    @user = User.find(params[:user_id])
+    @user = current_user
```

Comment:
- [ ] According to the project requirements, you should use **CanCanCan** for Post authorization in `Post Controller` bt loading `load_and_authorize_resource`.

Reply:
Hello @omar-labana, thank you for the review, but can you please explain more briefly what I should be implementing and what the issue is?

Reply:
@VSeray Sorry for not replying in time, I see that you have added the missing guard, good luck in the next milestones